test/system/kubernetes/fluentBitSetup.sh

```diff
@@ -0,0 +1,360 @@
+#!/usr/bin/env bash
```

Comment:
This file needs a copyright header. Find one in another `.sh` file and paste it here please.